src/Umbraco.Infrastructure/BackgroundJobs/Jobs/WebhookFiring.cs

```diff
@@ -90,20 +89,30 @@ public async Task RunJobAsync()
 
     private async Task<HttpResponseMessage?> SendRequestAsync(IWebhook webhook, string eventName, string? serializedObject, int retryCount, CancellationToken cancellationToken)
     {
-        using var httpClient = new HttpClient();
+        using HttpClient httpClient = _httpClientFactory.CreateClient(Constants.HttpClients.WebhookFiring);
 
-        var stringContent = new StringContent(serializedObject ?? string.Empty, Encoding.UTF8, MediaTypeNames.Application.Json);
-        stringContent.Headers.TryAddWithoutValidation("Umb-Webhook-Event", eventName);
-
-        foreach (KeyValuePair<string, string> header in webhook.Headers)
+        var request = new HttpRequestMessage(HttpMethod.Post, webhook.Url)
```

Comment:
Lets make sure this gets disposed correctly:
```suggestion
        using var request = new HttpRequestMessage(HttpMethod.Post, webhook.Url)
```

We can also dispose the `HttpReponseMessage` above (line 74) by adding a `using` to it 👍🏻